src/content/whats-new/2022/03/whats-new-03-08-io-partner-integrations.md

```diff
@@ -0,0 +1,33 @@
+---
+title: 'New integrations: CircleCI, Netlify, ReleaseIQ, MuleESB, Databricks, and Jira' 
+summary: 'Six partner quickstarts that help you start monitoring CI/CD, data analytics, and more even faster' 

```

Comment:
Let's add a comma or ellipses after the word "more"

Reply:
@alexiskjones, good catch! I just fixed this!